browser/eb_application.py

```diff
@@ -0,0 +1,357 @@
+"""Flask app route handlers"""
+
+import base64
+import json
+import os
+import pickle
+import re
+import sys
+import traceback
+
+import MySQLdb
+
+from flask import Flask, jsonify, render_template, request, redirect
+
+from helpers import is_trk_file, is_npz_file
+from caliban import TrackReview, ZStackReview
+import config
+
+# Create and configure the app
+application = Flask(__name__)  # pylint: disable=C0103
+app = application
+app.config.from_object("config")  # TODO: did this break by adding new env vars?
```

Comment:
Have we stress tested re: this todo?

Reply:
I am not sure exactly what that comment (Will's, I think) was referring to. Off the top of my head I think this step used to work by accessing environment variables from os, and the config.py was pretty different. Configuration no longer relies on any environment variables and is not broken so I think this is a non-issue.